libs/tokens/src/updaters/TokensListsUpdater/index.ts

```diff
@@ -27,7 +28,7 @@ const swrOptions: SWRConfiguration = {
   revalidateOnFocus: false,
 }
 
-const NETWORKS_WITHOUT_RESTRICTIONS = [SupportedChainId.SEPOLIA]
+const NETWORKS_WITHOUT_RESTRICTIONS = [SupportedChainId.SEPOLIA, SupportedChainId.GNOSIS_CHAIN]
```

Comment:
I think you might want to use `fix/disable-token-restrictions-on-gnosis` as base branch instead

Reply:
Fixed

Reply:
Thanks!